tests/routes/profile.test.js

```diff
@@ -0,0 +1,78 @@
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import faker from 'faker';
+import sinonchai from 'sinon-chai';
+import Model from '../../src/models';
+import app from '../../src';
+
+const { Users } = Model;
+chai.use(chaiAsPromised);
+chai.use(sinonchai);
+const profileUrl = '/api/v1/users/profile/';
+
+describe('Profile', () => {
+  const fakeId = '94189e3d-0379-4dd2-b03d-73fa8c14b3ab';
+  before(async () => {
+    await Users.destroy({ cascade: true, truncate: true });
+  });
+  let newUser;
```

Comment:
Declare all variables before the `before` hook. Also leave new lines around the before hook

Reply:
Done